python/paddle/fluid/transpiler/memory_optimization_transpiler.py

```diff
@@ -383,10 +381,13 @@ def memory_optimize(input_program, skip_opt_set=None, print_log=False, level=0):
 
       Note: it doesn't not support subblock nested in subblock.
 
-    :param input_program: Input Program
-    :param print_log: whether to print debug log.
-    :param level: If level=0, reuse if the shape is completely equal, o
-    :return:
+    Args:
```

Comment:
maybe you should add a testcase when skip_opt_set is not None.

Reply:
sure.